cvmfs/sync_union.h

```diff
@@ -233,12 +252,12 @@ class SyncUnionOverlayfs : public SyncUnion {
                            const std::string &filename);
   void ProcessCharacterDevice(const std::string &parent_dir,
                               const std::string &filename);
-  std::string UnwindWhiteoutFilename(const std::string &filename) const;
+  std::string UnwindWhiteoutFilename(const SyncItem &entry) const;
   std::set<std::string> GetIgnoreFilenames() const;
-  virtual void ProcessFile(SyncItem *entry);
+  virtual void ProcessFile(SyncItem &entry);
```

Comment:
[Done](https://github.com/reneme/cvmfs/commit/d258d78a77005a992cae6f2c4ec7a5bf0e9c56b7) even though I consider this nitpicking. Especially [here](https://github.com/reneme/cvmfs/commit/d258d78a77005a992cae6f2c4ec7a5bf0e9c56b7#diff-4d6b1fda58c9782181cc19b60ba1d3ebL298), where it takes a significant thinking-overhead to figure out that actually the `*entry` is a `const&`-parameter and therefore de-referenced, while the method right underneath is actually changing `entry`.